src/applications/edu-benefits/components/SchoolSelectField.jsx

```diff
@@ -224,7 +236,7 @@ export class SchoolSelectField extends React.Component {
 
 const mapStateToProps = (state, props) => {
   const currentPageNumber = selectCurrentPageNumber(state);
-  const facilityCodeSelected = props.formData ? props.formData : '';
+  const facilityCodeSelected = props.formData.facilityCode ? props.formData.facilityCode : '';
```

Comment:
This component already uses the `formData` as the source of truth for the currently selected facility code.